dagrunner/utils/visualisation.py

```diff
@@ -7,33 +7,65 @@
 """
 
 import base64
+import math
 import os
+import warnings
+import webbrowser
+from typing import Iterable, Union
+
+import networkx as nx
+
+from . import as_iterable, in_notebook
+
+# set web component path to relative local file when running pytest
+if os.environ.get("PYTEST_VERSION") is not None:
+    WEBCOMPONENT_PATH = "../../../visual/mermaid-table-standard.js"
+else:
+    WEBCOMPONENT_PATH = "https://cdn.jsdelivr.net/gh/MetOffice/dagrunner@vvisual_tooltip_table/visual/mermaid-table-standard.js"
```

Comment:
Approach to checking if running as pytest as per https://docs.pytest.org/en/stable/example/simple.html#detect-if-running-from-within-a-pytest-run